server/socket/socket.js

```diff
@@ -1,22 +1,23 @@
-const { io } = require("../app");
-const jwt = require("jsonwebtoken");
 const cookie = require('cookie');
+const jwt = require("jsonwebtoken");
 
-io.use((socket, next) => {
-  const cookies = cookie.parse(socket.handshake.headers.cookie);
-  if (cookies.token) {
-    const token = cookies.token;
-    const decodedToken = jwt.verify(token, process.env.SECRET);
-    socket.userId = decodedToken.userId;
-    next();
-  } else {
-    next("Unable to authenticate");
-  }
-})
-.on("connection", (socket) => {
-  console.log(`${socket.userId} connected`);
+module.exports = (io) => {
+  io.use((socket, next) => {
+    const cookies = cookie.parse(socket.handshake.headers.cookie);
+    if (cookies.token) {
+      const token = cookies.token;
+      const decodedToken = jwt.verify(token, process.env.SECRET);
```

Comment:
We might need to change this, because we're assuming here that the token is valid. If the token is expired, this just fails, so a callback is needed. You can take a look at my new PR once I put it up so we can have it consistent.

Reply:
I should do that in another pr since this was just to move everything around so that the circleci tests finally work